components/builder-core/src/access_token.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2017 Chef Software Inc. and/or applicable contributors
```

Comment:
Should this be 2018?

Reply:
I seem to recall we concluded not worrying about the copyright date and not needing to change the standard header to new dates - although I think part of this code may be from 2017 anyways.